src/autocomplete.js

```diff
@@ -580,13 +580,20 @@ class Autocomplete {
             this.tooltipNode = dom.createElement("div");
             this.tooltipNode.style.margin = 0;
             this.tooltipNode.style.pointerEvents = "auto";
+            this.tooltipNode.style.overscrollBehavior = "contain";
```

Comment:
Do we need this line?

Reply:
It is for doc-tooltips of autocomplete items, they are also scrollable.